compiler/src/dmd/dmodule.d

```diff
@@ -1525,7 +1528,7 @@ private const(char)[] processSource (const(ubyte)[] src, Module mod)
 
         if (buf.length & 1)
         {
-            mod.error("odd length of UTF-16 char source %llu", cast(ulong) buf.length);
+            .error(mod.loc, "%s `%s` odd length of UTF-16 char source %llu", mod.kind, mod.toPrettyChars, cast(ulong) buf.length);
```

Comment:
This caused a diagnostic regression - here, `mod.loc is Loc.initial`.

Before:
```
bom_error_UTF16.d: Error: module `bom_error_UTF16` odd length of UTF-16 char source 159
```

After:
```
Error: module `bom_error_UTF16` odd length of UTF-16 char source 159
```